src/controllers/movie.js

```diff
@@ -0,0 +1,72 @@
+const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
+
+const getAllMovies = async (req, res) => {
+  const movies = await prisma.movie.findMany ({
+    include: {
+        screenings: true,
+      },
+     });
+
+  return res.status(200).json({ movies });
+};
+
+const createMovie = async (req, res) => {
+  const { title, runtimeMins } = req.body;
+
+  const movie = await prisma.movie.create({
+    data: {
+      title: 'Top Gun',
+      runtimeMins:110,
+    },
+    include: {
+      screenings: true,
+    },
+  });
+
+  return res.status(201).json({ movie });
+};
+
+const getMovieById = async (req, res) => {
+    const { id } = req.params;
+
+    const foundMovie = await prisma.movie.findUnique({
+        where: {
+            id: Number(id),
+        },
+        include: {
+            screenings: true,
+        }
+    })
+    return res.status(200).json({ movie: foundMovie });
```

Comment:
Had you named your variable on line 33 movie you could have returned ({ movie }) instead of having to write it out like you have